test/e2e/storage/sanity.go

```diff
@@ -496,10 +492,12 @@ var _ = Describe("sanity", func() {
 			)
 
 			BeforeEach(func() {
+				// Worker nodes with PMEM.
 				nodes = make(map[string]nodeClient)
-				for i, addr := range workerSocatAddresses {
+				for i := 1; i < cluster.NumNodes(); i++ {
```

Comment:
if we start with one and check for less than NumNodes, is there a risk we don't reach last one?

Reply:
We skip the first one (index == 0), which is intentional here because we just want the "worker nodes with PMEM".

Reply:
ok, just wanted to be sure this is intentional